spec-main/api-app-spec.ts

```diff
@@ -674,25 +674,30 @@ describe('app module', () => {
 
   describe('getPath("logs")', () => {
     const logsPaths = {
-      'darwin': path.resolve(homedir(), 'Library', 'Logs'),
-      'linux': path.resolve(homedir(), 'AppData', app.name),
-      'win32': path.resolve(homedir(), 'AppData', app.name),
+      'darwin': path.resolve(homedir(), 'Library', 'Logs', 'Electron'),
+      'linux': path.resolve(homedir(), 'AppData', 'Electron', 'logs'),
+      'win32': path.resolve(homedir(), 'AppData', 'Electron', 'logs'),
     }
 
+    after(() => {
+      const osLogPath = (logsPaths as any)[process.platform]
+      fs.rmdirSync(osLogPath)
```

Comment:
I don't think we want to call this unconditionally. It looks like this will remove the directory for any human who happens to run this test, as well.